From 91663ceb3259ea2fcc15365323f9a86fba10634a Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Mon, 5 Jan 2026 20:28:08 +0100
Subject: [PATCH 094/N] makesetup: make sure a library path is passed when
 building modules

This is fallout from LIBPYTHON not inluding -L. anymore
---
 Modules/makesetup | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Modules/makesetup b/Modules/makesetup
index 039d361..2ffc3dd 100755
--- a/Modules/makesetup
+++ b/Modules/makesetup
@@ -276,7 +276,7 @@ sed -e 's/[ 	]*#.*//' -e '/^[ 	]*$/d' |
 				;;
 			esac
 			rule="$file: $objs \$(MODULE_${mods_upper}_LDEPS)"
-			rule="$rule; $ccld $objs $libs \$(LIBPYTHON) -o $file"
+			rule="$rule; $ccld $objs $libs \$(BLDLIBRARY) -o $file"
 			echo "$rule" >>$rulesf
 		done
 	done
